pureskillgg_dsdk/ds_io/reader_fs.py

```diff
@@ -79,4 +79,4 @@ def _get_file_path(self, channel: Union[str, Dict]) -> str:
 def add_prefix(key, prefix, /) -> str:
     if prefix is None:
         return key
-    return os.path.join(prefix, key)
+    return os.path.join(*[*prefix.split("/"), key])
```

Comment:
```suggestion
    return os.path.join(*[*prefix.split(os.sep), key])
```
I think this breaks windows paths as prefixes, which we should support... probably

Reply:
No the idea w/ this is to make `prefix` like `key_prefix` meaning using `/` is correct. This should convert the `/` to the os sep.